src/modular/monty_form.rs

```diff
@@ -8,32 +8,32 @@ mod pow;
 mod sub;
 
 use super::{
+    const_monty_form::{ConstMontyForm, ConstMontyParams},
     div_by_2::div_by_2,
     reduction::montgomery_reduction,
-    residue::{Residue, ResidueParams},
     Retrieve,
 };
 use crate::{Limb, NonZero, Uint, Word, Zero};
 use subtle::{Choice, ConditionallySelectable, ConstantTimeEq, CtOption};
 
 /// Parameters to efficiently go to/from the Montgomery form for an odd modulus provided at runtime.
 #[derive(Clone, Copy, Debug, PartialEq, Eq)]
-pub struct DynResidueParams<const LIMBS: usize> {
+pub struct MontyParams<const LIMBS: usize> {
     /// The constant modulus
     modulus: Uint<LIMBS>,
-    /// Parameter used in Montgomery reduction
-    r: Uint<LIMBS>,
-    /// R^2, used to move into Montgomery form
+    /// 1 in Montgomery fomr
```

Comment:
Typo in "form", but maybe the comments should also mention this is `R`?

```suggestion
    /// 1 in Montgomery form (a.k.a. `R`)
```